nntrainer/graph/network_graph.cpp

```diff
@@ -316,12 +312,15 @@ int NetworkGraph::realizeActivationType(Layer &current) {
   std::shared_ptr<Layer> layer = nntrainer::createLayer(ActivationLayer::type);
   layer->setActivation(act);
 
+  ensureName(layer, current.getName());
+
   if (current.getType() == TimeDistLayer::type) {
+    std::string unit_str = layer->getName();
+    layer->setName(layer->getName() + "_unit");
```

Comment:
please ensure that this name does not already exist.

Reply:
> please ensure that this name does not already exist.

well.. I guess, layer->getName() is unique.. then this is unique. And the uniqueness is checked already.

Reply:
`layer->getName()` is unqiue, but layer->getName() + 'xyz' can exist. There is no check to verify this.

Reply:
This will be updated in #1144